### ISSUE TITLE
[Chore] Use Active Model Errors in Keyword

### ISSUE THREAD

**Issue by olivierobert** [ayerdines/nimble-scraper#8]
## Issue

In the `Keyword` model, the class method `import_from_csv` returns a tuple:

https://github.com/ayerdines/nimble-scraper/blob/0a4da3f79cb6e78657dc84e120e31700801f90a7/app/models/keyword.rb#L14-L21

## Expect

Active Record-backed models can leverage [Active Model errors](https://api.rubyonrails.org/v7.0.4/classes/ActiveModel/Errors.html)

```ruby
if keywords.count > 100
   [:unprocessable_entity, ]
   errors.add(:file, :invalid, message: "File contains more than 100 keywords.'") 
```

At the same time, it might make more sense to move this validation to a dedicated PORO that uses Active Model since the validation seems a bit out of place:

```
class KeywordCsv 
  include ActiveModel::Model

   attr_reader :file

   # add validate code
end
```

**Comment by ayerdines** [comment]
Yes, you're correct. I think a better approach would be to create a service that handles the CSV import. What do you think @olivierobert?
